Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook

```diff
@@ -0,0 +1,1220 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## Workspace details"
+      },
+      "name": "Heading1"
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "paragraph",
+        "links": [
+          {
+            "id": "bac30f99-eaf9-4c7f-8705-cc263b8e1e88",
+            "cellValue": "https://aka.ms/BCDRAuditReportTemplates",
```

Comment:
Please change link to - https://learn.microsoft.com/en-us/azure/backup/azure-policy-configure-diagnostics